conditional-kotlin/src/main/kotlin/com/linecorp/conditional/kotlin/ConditionalExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.linecorp.conditional.kotlin
+
+import com.linecorp.conditional.ComposedCondition
+import com.linecorp.conditional.Condition
+
+infix fun Condition.and(condition: Condition): ComposedCondition = this.and(condition)
+
+infix fun Condition.or(condition: Condition): ComposedCondition = this.or(condition)
+
+infix fun Condition.nor(condition: Condition): ComposedCondition = this.nor(condition)
```

Comment:
What is the role of this `nor` method??

Reply:
This is a mistake 😓 
Looking at the `@MethodSource("NOR")` code of `ConditionTest`, I thought that there is a `NOR` operator.

Fixed it, [1e71453](https://github.com/line/conditional/pull/4/commits/1e71453ec98a2b86f06acae279818c5eecff789f) 🙂

Reply:
Oh, I see. Could you add the `not` operator as well?

Reply:
I wonder if the infix function for `not` is appropriate.

For example, if I declare the infix function like this...
```kotlin
infix fun Condition.not(condition: Condition): Condition = this.not(condition)
```

Then I can use it like this... but it's a little bit awkward.
```kotlin
val a = Condition.trueCondition() not Condition.falseCondition() // ...?
```

It would be nice if it could be used like this, but it is impossible due to the limitations of the syntax.
```kotlin
val a = not Condition.trueCondition() // compile error
```

Am I getting the points right?

Reply:
I mean we can use `not` it like below. Maybe [this reference](https://kotlinlang.org/docs/operator-overloading.html) can help :) 
```kotlin
!Condition.trueCondition()
```

Reply:
I wrote the not operator function with simple test codes.  (565b242)

Since not operator does not return `ComposedCondition`, I separated it into a separate class and wrote it simply.
Please feel free to comment on naming, location, etc 🙂

Reply:
That's great! We almost done. Could you add the copyright header? All source files must begin with the following copyright header. It is convenient to set it in the IDE.

```
Copyright $today.year LINE Corporation

LINE Corporation licenses this file to you under the Apache License,
version 2.0 (the "License"); you may not use this file except in compliance
with the License. You may obtain a copy of the License at:

  https://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
License for the specific language governing permissions and limitations
under the License.
```